spec/spec_helper_acceptance.rb

```diff
@@ -1,6 +1,6 @@
 require 'voxpupuli/acceptance/spec_helper_acceptance'
 
-ENV['BEAKER_setfile'] ||= 'centos8-64{hostname=centos8-64.example.com}'
+ENV['BEAKER_setfile'] ||= 'centos9-64{hostname=centos9-64.example.com}'
```

Comment:
won't this be overwritten by modulesync anyway?

Reply:
Touche.